resources/js/components/FormField.vue

```diff
@@ -86,9 +86,14 @@ export default {
         },
 
         deleteImage(index) {
-            axios.delete('/nova-vendor/array-images/delete/'+this.images[index].name);
-            this.images.splice(index, 1);
-            this.value = JSON.stringify(this.images);
+            const { saved_path } = this.images[index]
+            const params = { disk: this.field.disk }
+            axios.delete('/nova-vendor/array-images/delete/' + encodeURIComponent(saved_path), { params })
+              .then(() => {
+                this.images.splice(index, 1);
+                this.value = JSON.stringify(this.images);
+              })
+              .catch(console.warn)
```

Comment:
Because it is not an error in the frontend, and indeed this is not the best way to handle the error, actually it should show a toast or mark the file in some way to let the user know that maybe communication with the backend was temporarily not possible (timed-out), the file could not be found, the path was wrong, etc.